pcsx2/System.cpp

```diff
@@ -358,6 +358,27 @@ namespace HostMemoryMap {
 
 /// Attempts to find a spot near static variables for the main memory
 static VirtualMemoryManagerPtr makeMainMemoryManager() {
+	const wxString memoryManagerName = "Main Memory Manager";
+	
+	auto tryMakeAt = [&memoryManagerName](uptr base) -> VirtualMemoryManagerPtr {
+		auto mgr = std::make_shared<VirtualMemoryManager>(memoryManagerName, base, HostMemoryMap::Size, /*upper_bounds=*/0, /*strict=*/true);
+		if (mgr->IsOk())
+			return mgr;
+		return nullptr;
+	};
+
+	// Historically, the base address has always been 0x20000000 for x86 builds. Many cheat tables and third party
+	// tools relies to this specific address, therefore try to initialize the memory manager using this base
+	// address first. This does not guarantee it, but it prioritise the legacy memory address.
+	// This is only valid for x86 builds, as x64 have a high chance of failing an assertion by using this
+	// specific address. x64 builds currently uses 0x60000000.
```

Comment:
Position in x64 depends on where your linker decides to put PCSX2, and will vary by platform, so I wouldn't put it in the comments here

e.g. on my computer (macOS 10.15) it's at 0x140000000

Reply:
Comment removed!